build/yamls/flow-visibility.yml

```diff
@@ -4829,6 +4784,44 @@ spec:
 ---
 apiVersion: apps/v1
 kind: Deployment
```

Comment:
I thought we were going to use a new container within the same Deployment, and not a new Deployment?

There is value in keeping the number of Pods down.

Reply:
As We consider to deploy the ClickHouse cluster in future which may have multiple ClickHouse pods, but we only need one monitor for the whole ClickHouse cluster. Thus we split the monitor deployment from the ClickHouse.

Reply:
The fact is that at the moment, there is a single replica so there is no reason not to go with the simple solution.

Looking at the ClickHouse operator in more details it is very easy to add a container: https://github.com/antrea-io/antrea/blob/55e6797ce23d5f79210797c2f26c3a867ac67134/build/yamls/flow-visibility.yml#L5004. And because the ClickHouse servers are run as a StatefulSet, and not as a Deployment, if we ever have more than 1 replica, it will be easy to run the monitor for the first replica only (only one container will do the work, the others will not do anything).

Reply:
Got it. Thanks Antonin, I updated the code to move the monitor to the ClickHouse pod.